system-addon/test/unit/lib/ActivityStreamMessageChannel.test.js

```diff
@@ -80,32 +80,35 @@ describe("ActivityStreamMessageChannel", () => {
         mm.createChannel();
         assert.notCalled(global.AboutNewTab.override);
       });
-      it("should simulate init for existing ports", () => {
-        sinon.stub(mm, "onActionFromContent");
-
-        RPmessagePorts.push({
-          url: "about:monkeys",
-          loaded: false,
-          portID: "inited"
-        });
-        RPmessagePorts.push({
-          url: "about:sheep",
-          loaded: true,
-          portID: "loaded"
+      describe("#dispatchInitLoadEvent", () => {
```

Comment:
This should be at the same level as `#createChannel` and would avoid indenting everything.